lib/bolt/logger.rb

```diff
@@ -81,7 +80,7 @@ def self.console_layout(color)
 
     def self.default_layout
       Logging.layouts.pattern(
-        pattern: '%d %-6l %c: %m\n',
+        pattern: '%d %-6l [%T] [%c] %m\n',
```

Comment:
I like this change a lot! It looks like the applicator still needs to name the current thread, though:

```
2020-08-19T09:02:50.431928 INFO   [] [Bolt::Executor] Compiling manifest block on ["server"]
```

Reply:
Updated, and I set names for the TPEs so the threads will show up as `exec-worker-1` and `apply-worker-1`.